clients/geth/specular/sbin/init_clef.exp

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/expect
+
+set CLEF_PW "unsafe-password"
+
+set PW_FILE [open "../data/password.txt"]
+set ADDR_PW [read $PW_FILE]
+
+spawn ./init_clef.sh
+
+# init clef masterseed
+expect ">"
+send "ok\n"
+
+expect "Password: "
+send "$CLEF_PW\n"
+
+expect "Repeat password: "
+send "$CLEF_PW\n"
+
+# set account password for 0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266
```

Comment:
we shouldn't hardcode these anywhere (even in comments) -- you can refer to them as "sequencer" etc instead

same goes for below and in `init_clef`